package.json

```diff
@@ -11,7 +11,7 @@
     "build": "ember build",
     "build-prod": "ember build -prod",
     "clean": "rm -rf node_modules/ bower_components/ dist/ tmp/",
-    "clean-deep": "npm run clean && npm cache clean && bower cache clean",
```

Comment:
why?

Reply:
npm 5  guarantees cache to be in good state